modules/services/mapillary.js

```diff
@@ -32,7 +33,46 @@ let _mlyShowFeatureDetections = false;
 let _mlyShowSignDetections = false;
 let _mlyViewer;
 let _mlyViewerFilter = ['all'];
+let _dataset;
 
+const rapidTypes = ['object--support--utility-pole', 'object--street-light', 'object--bench' ,'object--bike-rack', 'object--fire-hydrant' ];
+const tag_dict = {
+    'object--fire-hydrant': {
+        emergency: 'fire_hydrant'
+    },
+    'object--support--utility-pole': {
+        power: 'pole',
+        'man_made': 'utility_pole'
```

Comment:
most of the keys in here and not in quotes but 'man_made' is -- not sure which is correct but should that be fixed?

Reply:
I'd remove the quotes from the member identifier, but leave them in for the string type values. 

e.g. `'emergency' : 'fire_hydrant'`

to `emergency: 'fire_hydrant'`